src/UI/Component/Input/Field/Factory.php

```diff
@@ -205,4 +205,46 @@ public function dependantGroup(array $inputs);
 	 * @return    \ILIAS\UI\Component\Input\Field\Checkbox
 	 */
 	public function checkbox($label, $byline = null);
+
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     A textarea is intended for entering multi-line texts.
+	 *   composition: >
+	 *      Textarea fields will render an textarea HTML tag.
+	 *   effect: >
+	 *      Textarea inputs are NOT restricted to one line of text.
+	 *   rivals:
```

Comment:
JF "A textarea counts the amount of character input by user and displays the number."